ui/src/pages/ApplicationDetailsPage/components/BranchesList/BranchCard.tsx

```diff
@@ -0,0 +1,77 @@
+import { Card, CardActionArea, CardContent, Skeleton, Stack, Typography } from '@mui/material'
+import { ArtifactListItemReducedDTO } from '../../../../../autogenerated/client/backend'
+import { useNavigate } from 'react-router-dom'
+import moment from 'moment'
+import { Commit, LayersOutlined, SvgIconComponent, SyncOutlined } from '@mui/icons-material'
+import GitBranch from '../../../../components/GitBranch/GitBranch'
+import InstallDialog from '../InstallDialog'
+import { useState } from 'react'
+import HintIcon, { LoadingHintIcon } from './HintIcon'
+
+export default function BranchCard({ branch, version, artifact }: 
+{ branch: { name: string, pattern: string }, version: string, artifact: ArtifactListItemReducedDTO }) {
+  const [ installDialogOpen, setInstallDialogOpen ] = useState<boolean>(false)
+
+  const navigate = useNavigate()
+
+  function onInstall() {
+    setInstallDialogOpen(false)
+    navigate('/workloads')
+  }
+
+  return (
+    <>
+      <Card 
+        variant='outlined' 
+        sx={ { 
+          maxWidth: 'lg', 
+          '&:hover': { borderColor: 'primary.main', boxShadow: 2 } 
+        } }>
+        <CardActionArea onClick={ () => setInstallDialogOpen(true) }>
+          <CardContent>
+            <Stack direction='row' justifyContent='space-between'>
+              <Stack direction='column' alignItems='start' spacing={ 1 }>
+                <Typography variant='h4'>Version { version }</Typography>
+                <Stack direction='row' justifyContent='start' alignItems='center' spacing={ 2 }>
+                  <HintIcon tooltip='Branch' icon={ GitBranch as SvgIconComponent } text={ branch.name } />
+                  <HintIcon tooltip='Chart version' icon={ LayersOutlined } text={ `${ artifact.version }-${ artifact.revision }` } />
+                  <HintIcon tooltip='Last updated' icon={ SyncOutlined } text={ moment(artifact.registered_at).fromNow() } />
+                  <HintIcon tooltip='Short digest' icon={ Commit } text={ artifact?.digest.value.substring(0, 7) } />
```

Comment:
I think using the commit icon here is a bit misleading, As the short digest also has the format of a Git commit id. Do you think we can change it to something else?

Reply:
LGTM